membership.workflows.yml

```diff
@@ -16,3 +16,35 @@ membership_default:
       label: Restart
       from: [expired]
       to: active
+
+membership_term_default:
+  id: membership_term_default
+  label: Default Membership Term Workflow
+  group: membership_term
+  states:
+    pending:
+      label: Pending
+    active:
+      label: Active
+    expiring:
+      label: Expiring
```

Comment:
Interesting addition of "Expiring." I don't see any code (yet) that would automate the transition from Active to Expiring (or any other transitions, actually) but is that the idea? Similar with the Pending state... that seems to be an implementation detail, not all sites will have a pending state or need to implement it.

I think the "expiring" state could mean different things to different people, or not apply at all... do you think breaking this out into a second module-provided workflow would make sense? And keep the "default-default" to Active and Expired?